### PR TITLE
Issue 617

### LINKED ISSUE #617: Bug: admins of group are kicked out of group and have to re-join, and also show up as duplicates
If I create a group and make a new member an admin, that member suddenly gets kicked out of the group and has to rejoin. They also can't start a hangout.

This doesn't happen when I make the new member a moderator.

Possibly relevant error:
```
Exception in setTimeout callback: Error: Can't select in removed DomRange
    at Blaze._DOMRange.DOMRange.$ (http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:721:11)
    at Blaze.TemplateInstance.$ (http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:3574:25)
    at http://localhost:3000/app/app.js?hash=6475462e89ec2ac0e2bef514363586fbca3b066a:4825:16
    at Meteor.EnvironmentVariable.withValue (http://localhost:3000/packages/meteor.js?hash=27829e936d09beae3149ecfbf3332c42ccb1596f:1077:17)
    at http://localhost:3000/packages/meteor.js?hash=27829e936d09beae3149ecfbf3332c42ccb1596f:464:45
    at http://localhost:3000/packages/meteor.js?hash=27829e936d09beae3149ecfbf3332c42ccb1596f:1105:22
```

### PR DESCRIPTION
**Pull request by lpatmo**
Fixes #617.

### CODE REVIEW

File: server/hangouts/methods.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ Meteor.methods({
 
       return true;
 
-    }else if(Roles.userIsInRole(loggedInUser._id,['admin','moderator'], 'CB')){
+    }else if(Roles.userIsInRole(loggedInUser._id,['owner', 'admin','moderator'], 'CB')){
```
Comment (distalx) [inline server/hangouts/methods.js]: Please undo this change for the reason mentioned above.

File: server/hangouts/methods.js [hunk 2]
```diff
@@ -205,7 +205,7 @@ Meteor.methods({
   },
   getHangout: function(hangoutId) {
     check(hangoutId, String);
-    if (Roles.userIsInRole(this.userId, ['admin','moderator'], 'CB')) {
+    if (Roles.userIsInRole(this.userId, ['owner', 'admin','moderator'], 'CB')) {
```
Comment (distalx) [inline server/hangouts/methods.js]: Please undo this change for the reason mentioned above.

File: server/hangouts/methods.js [hunk 3]
```diff
@@ -352,7 +352,7 @@ Meteor.methods({
 
       return true;
 
-    }else if(Roles.userIsInRole(loggedInUser._id,['admin','moderator'], 'CB')){
+    }else if(Roles.userIsInRole(loggedInUser._id,['owner', 'admin','moderator'], 'CB')){
```
Comment (distalx) [inline server/hangouts/methods.js]: Please undo this change for the reason mentioned above.

**Comment by lpatmo** [comment]
Thank you so much for the feedback, @distalx!

**Pull request merged**